lua/autorun/server/sv_property_restrictions.lua

```diff
@@ -0,0 +1,17 @@
+--/\-- Requiring / Including
+
+
+
+--/\-- Planning
+
+--/\-- Variables
+
+--/\-- Functions
+
+
+--/\-- Hooks
+hook.Remove("CanProperty", "persist_property")
+
+hook.Add( "CanProperty", "persist_property", function( ply, property, ent )
+	if ( !ply:IsAdmin() && property == "persist" ) then return false else return true end

```

Comment:
This line is a little dense and a little long. We don't want to always return something here because other hooks may need to run (returning in a hook automatically halts the hook chain, meaning no other hooks after this one can run), so we only want to return when we want to alter the behavior.

For example;
```suggestion
    if not ply:IsAdmin and property == "persist" then return false end
```